### PR TITLE
Create To-Do List

### PR DESCRIPTION
**Pull request by AaryaNale**
This is a simple To-Do List Application: Manage tasks with ease using this Python-based tkinter app. Add, toggle, and view tasks in a user-friendly graphical interface. Stay organized effortlessly.

### CODE REVIEW

**Comment by AaryaNale** [comment]
Please review and suggest changes to be made, if any.
Thanks.

File: To-Do-List/To_Do.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Create a button to open the instruction window
+instruction_button = tk.Button(root, text="Instructions", command=open_instruction_window)
+instruction_button.pack(pady=10)
+
+# Create a listbox to display tasks
+listbox = tk.Listbox(root, selectmode=tk.SINGLE)
+listbox.pack(pady=10)
+
+# Create an entry field to add tasks
+entry = tk.Entry(root)
```
Comment (FerdausPolok) [inline To-Do-List/To_Do.py]: I'm not 100% sure but is this the input field we see on our GUI? If yes can we somehow sanitize it?

**Review by FerdausPolok — CHANGES_REQUESTED** [review]
Hello! Thanks for your contribution here.
One little change request from my end: can we not accept any empty (ie. "    ")? We may show the user a toast message or maybe just freeze the add task button until we have a valid task title inside our input field.
https://github.com/FerdausPolok/dev-mates/assets/68804751/340b4512-2f1e-4c8c-9d8d-423e59f50a81

Also, it's always a good practice to keep our brances separate. I see you did pushed your code to your main branch. Thanks.

File: To-Do-List/To_Do.py (hunk 1 above)
Comment (AaryaNale) [reply To-Do-List/To_Do.py]: Okay, so we dont want it to take empty tasks, right? 
I'll get on working on it

Comment (FerdausPolok) [reply To-Do-List/To_Do.py]: Yeap that would be nice.

**Comment by AaryaNale** [comment]
Improved the code. Do check it out and let me know if something else needs to be added.
Thanks.

**Review by FerdausPolok — APPROVED** [review]
LGTM 🚀
Optional but we can add a placeholder to our input field for better user experience. Like: Add task title here/ Add your task here/ Your task here...

**Comment by AaryaNale** [comment]
Please go through the changes @FerdausPolok

**Review by FerdausPolok — APPROVED** [review]

**Comment by AaryaNale** [comment]
Up to merge maybe? @FerdausPolok

**Pull request merged**